ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -1526,6 +1526,40 @@ def test_sosreport_without_healthcheck_installed(self, remove_healthcheck):
             ]
         )
 
+    @pytest.fixture
+    def change_pwd_plugin_default(self):
+        """
+        Fixture to change the password plugin feature
+        to AllowNThash and change it to default
+        """
+        self.master.run_command(
+            ["ipa", "config-mod", "--ipaconfigstring=AllowNThash"]
+        )
+        yield
+        self.master.run_command(
+            ["ipa", "config-mod",
+             "--ipaconfigstring={AllowNThash, 'KDC:Disable Last Success'}"]
+        )
+
+    def test_krbLastSuccessfulAuth_warning(self, change_pwd_plugin_default):
+        """
+        This test checks that warning message is displayed
+        when password plugin feature is modified to
+        AllowNThash
+        """
+        err_msg = (
+            "Last Successful Auth is enabled. "
+            "It may cause performance problems."
+        )
+        returncode, data = run_healthcheck(
+            self.master, "ipahealthcheck.ipa.config",
+            "IPAkrbLastSuccessfulAuth",
```

Comment:
This check is available in ipa-healthcheck 0.18 only.
As the tests are executed on f41 also (with 0.17), they have to be skipped when healthcheck version is too old.
Add something like
```
        version = tasks.get_healthcheck_version(self.master)
        if parse_version(version) < parse_version("0.18"):
            pytest.skip("Check does not exist in ipa-healthcheck < 0.18")
```
at the beginning of the test.